.coveragerc

```diff
@@ -13,6 +13,7 @@ omit =
     homeassistant/components/abode/*
     homeassistant/components/acer_projector/switch.py
     homeassistant/components/actiontec/device_tracker.py
+    homeassistant/components/adguard/*
```

Comment:
This will also disable coverage for config_flow - maybe it should be enabled for that file for the integration scale?

Reply:
I agree about the weirdness, but that is how it is done. Judged by history and current state of other integrations.

If this change is wished for, I suggest doing it repo wide on all integrations.

Reply:
It is done for ESPHome. I was instructed to do so when I created that integration.

Reply:
I've asked about it before, did not get a positive or definitive answer. It is not documented and some tests where actually masked during recent changes.

Considering that, I have to assume those tests on integration level are not wished to be included in the coverage.

I'm sorry to hear you had a different experience. I'm not willing to make this change unless it has been solved globally.

Reply:
This is part of the PR template, introduced in #23520:

![image](https://user-images.githubusercontent.com/1444314/58743638-3f5a9e00-83ea-11e9-832e-cd0744bb0b83.png)

Reply:
Check @balloob, agreed.
Updated `.coveragerc`.